### DISCUSSION TITLE
A bug in version 3.12.1

### DISCUSSION THREAD

**Discussion by zhubib1234** [javalent/obsidian-leaflet#71] [Q&A]
The distance in the lower left corner cannot be displayed properly.

**Comment by valentine195** [comment]
Can you give me an example of what you're seeing? It looks good to me:

![Untitled](https://user-images.githubusercontent.com/38669521/119657151-88873200-bdf9-11eb-89fe-6803f1f031d5.png)

**Comment by zhubib1234** [comment]
![0](https://user-images.githubusercontent.com/84388916/119663161-b05faf80-be64-11eb-9f11-b739a1f986a2.png)

**Reply by valentine195** [reply]
Okay, my guess: this is an image map and the scale on your map is off, causing that line to be less than 0.1 meters.

Does the distance box _ever_ display a distance above 0 meters?

**Reply by zhubib1234** [reply]
I see. thanks！
